port/osx/omrosdump.c

```diff
@@ -95,10 +95,97 @@ static uint8_t seg_data_buffer[SEG_DATA_BUFFER_SIZE];
 static char corefile_name[PATH_MAX];
 static int corefile_fd = -1;
 
+static kern_return_t setup_msg_port(mach_port_t *);
+static kern_return_t send_port_msg(mach_port_t, mach_port_t);
+static kern_return_t recv_port_msg(mach_port_t, mach_port_t *);
```

Comment:
I think it would be helpful to name the arguments to clarify the meaning of the parameters.
If a function is not used before its definition, the forward declaration could simply be removed.